projects/components/src/table/table.component.ts

```diff
@@ -273,13 +273,17 @@ export class TableComponent
     map(params => this.getPageData(params))
   );
 
+  public isTableFullPage: boolean;
+
   public dataSource?: TableCdkDataSource;
 
   public constructor(
     private readonly changeDetector: ChangeDetectorRef,
     private readonly navigationService: NavigationService,
     private readonly activatedRoute: ActivatedRoute
   ) {
+    this.isTableFullPage = this.display === TableStyle.FullPage;
```

Comment:
this check should be in ngOnChanges since it could technically change (although I admit, unlikely to)

Reply:
I omitted it there because TableStyle won't change across digest cycles. Will add it there.